lib/ansible/modules/files/acl.py

```diff
@@ -27,12 +27,13 @@
 description:
     - Sets and retrieves file ACL information.
 options:
-  name:
+  path:
     required: true
     default: null
     description:
       - The full path of the file or object.
```

Comment:
In this case both name and path used to work, they just switched cases.
But in other cases you are right.